projects/kit/components/input-number/input-number.component.ts

```diff
@@ -53,6 +55,10 @@ import {TUI_INPUT_NUMBER_OPTIONS, TuiInputNumberOptions} from './input-number.op
 
 const DEFAULT_MAX_LENGTH = 18;
 
+export const TUI_NUMBER_VALUE_TRANSFORMER = new InjectionToken<
```

Comment:
nit: maybe move the token to a separate file? (similar as data-value-transformers)

Reply:
I did that initially but this token is literally only needed here, while date-value-transformers are used in multiple components.